modules/core/04-channel/types/channel.go

```diff
@@ -7,6 +7,8 @@ import (
 	"github.com/cosmos/ibc-go/v8/modules/core/exported"
 )
 
+const MaximumVersionLength = 8192 // maximum length of the version in bytes
```

Comment:
this is just a ballpark figure yea? We have no information on the typical length of versions?

Reply:
Yes, it's a ballpark figure. Typically the version strings have been short so far, like `ics20-1` or `{"fee_version":"ics29-1", "app_version":"ics20-1"}`, at least for our use cases.